src/main/java/com/google/flourbot/api/DriveCloudSheet.java

```diff
@@ -12,49 +12,66 @@
 import java.util.List;
 
 public class DriveCloudSheet implements CloudSheet {
-  Sheets sheetsService;
-  String spreadsheetId;
+  private String valueInputOption = "USER_ENTERED";
+  private String insertDataOption = "INSERT_ROWS";
+  private Sheets sheetsService;
+  private String spreadsheetId;
 
   public DriveCloudSheet(String documentId, Sheets documentService) {
     this.spreadsheetId = documentId;
     this.sheetsService = documentService;
   }
 
-  // TODO: replace hardcoded range with calculated range (so far values must be length 3)
-  public void appendRow(List<String> values) throws IOException, GeneralSecurityException {
+  private String getRange(int numEntryColumns, int numSkipColumns) {
+    // Returns range for sheets request by calculating the alphabetic name of
+    // the start and end column based on the number of columns filled and pre-skipped
+
+    String startColumn = this.toAlphabetic(numSkipColumns);
```

Comment:
no need for "this."

Reply:
When do we include or omit "this" ? What are the best practices